packages/wallet/README.md

```diff
@@ -17,11 +17,9 @@ but for those with more specific needs, individual components can be imported.
 
 ```javascript
 const {
-
     Wallet,
-
-    verifyMessage
-
+    verifyMessage,
+    verifyTypedData
```

Comment:
I think that we do not support `verifyTypedData`. We are automatically throwing not supported. We should remove it altogether